### DISCUSSION TITLE
Out of curiousity, can the build output show all paths?

### DISCUSSION THREAD

**Discussion by simpel** [vercel/next.js#52382] [App Router]
Hi! 

When we build the site we get this nice summary of our site, see below. But can we get the output to show all generated paths? Or even better, can I export all the paths to something like excel easily?

![Screenshot 2023-07-07 at 12 33 42](https://github.com/vercel/next.js/assets/232024/9da16b46-9c43-4b26-9068-9b713951637c)

**Comment by icyJoseph** [comment]
Mmm I think there's some hard coded rules when printing those routes. 

I don't know if there's a good way to do this though, perhaps the `.next/prerender-manifiest.json`?
